emissionsapi/download.py

```diff
@@ -1,30 +1,45 @@
 """Module to filter and download the data from the ESA and store it locally.
 """
 
-import os
-import urllib3
+from sentinelsat import SentinelAPI
 
 from emissionsapi.config import config
+from emissionsapi.country_bounding_boxes import country_bounding_boxes
 import emissionsapi.logger
 
 # Logger
 logger = emissionsapi.logger.getLogger('emission-api.download')
 
 storage = config('storage') or 'data'
 
+user = 's5pguest'
+password = 's5pguest'
```

Comment:
I know the ESA is doing the same and writing this user and password on their website, but maybe you should add a comment here, so the next one changing this password to something reasonable does not simply paste it here to :smile: